### PR TITLE
[Vilnius] Add speakers

### PR DESCRIPTION
**Pull request by plaindocs**
WIP

- [x] images
- [x] waiting on one speaker confirmation 6/5/19
- [x] add speaker

### CODE REVIEW

File: docs/_data/2019.vilnius.speakers.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+Nowadays, most digital products are created by cross-functional teams of engineers, designers, business analysts and marketing managers, all of whom have to read, understand and sometimes even contribute to the documentation. Moreover, in every area of business, people deal with having to read knowledge bases, tutorials and technical documents. That’s why writing documentation is a very hard task to accomplish. You have to make it both understandable and approachable to a wider audience. 
+
+So far, many technical documents are still very complicated to the point of being almost impossible to understand for non-technical people. What’s more, many companies need to create two types of documents for different target audiences, which in many cases doesn’t make sense. In my talk, I’m going to show you how to write technical documents that are clear and useful to people with and without a technical background while still being full of valuable content and details."
+  slug:
```
Comment (plaindocs) [inline docs/_data/2019.vilnius.speakers.yaml]:
```suggestion
  slug: technically-you-can-creating-technical-content-for-all-audiences-malzgorita-trojanowska
```

File: docs/_data/2019.vilnius.speakers.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+So far, many technical documents are still very complicated to the point of being almost impossible to understand for non-technical people. What’s more, many companies need to create two types of documents for different target audiences, which in many cases doesn’t make sense. In my talk, I’m going to show you how to write technical documents that are clear and useful to people with and without a technical background while still being full of valuable content and details."
+  slug: 
+  speakers:
+  - name: Małgorzata Trojanowska
+    slug: malzgorita-trojanowska
+    twitter: 
+    website: https://www.linkedin.com/in/malgorzata-trojanowska-403671155/
+  title:  "Technically you can: creating technical content for all audiences."
```
Comment (plaindocs) [inline docs/_data/2019.vilnius.speakers.yaml]:
```suggestion
  title:  "Technically you can: creating technical content for all audiences"
```

**Review by plaindocs — COMMENTED** [review]
Good stuff @aidiss !

**Comment by plaindocs** [comment]
https://deploy-preview-883--dentist-canary-64055.netlify.com/conf/vilnius/2019/speakers/

OK, as far as I'm concerned we can merge this, and add the missing photo later.

**Review by thatdocslady — APPROVED** [review]
Looks great! Going to merge this and prepare the blog post PR :)

**Pull request merged**